crates/stackable-operator/CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ All notable changes to this project will be documented in this file.
 
 - Update `kube` to `1.1.0` ([#1049]).
 
+### Removed
+
+- Removed `last_update_time` from CRD ClusterCondition status ([#1054]).
```

Comment:
note: This should be marked as breaking.

Reply:
I tested it and dont feel its not breaking. Applying the new CRD and installing the new operator version simply removes the `last_updated_time` from the CRD.

Reply:
It might not be breaking for us, but some users might use this value. As such, a removal is a breaking change for them.

Reply:
added the breaking hint https://github.com/stackabletech/operator-rs/pull/1054/commits/244bfbbd1be6e6dd2bcd38454246da9ebcf0177e